### ISSUE TITLE
Inline code in epytext_demo

### ISSUE THREAD

**Issue by Codeberg-AsGithubAlternative-buhtz** [twisted/pydoctor#567]
https://github.com/twisted/pydoctor/blob/6e2febe63a8ed0f2e316009f34721dc1778661e7/docs/epytext_demo/__init__.py#L170

I'm not sure that is why I ask instead of creating an PR.

**Comment by tristanlatr** [comment]
Can you provide a example docstring and a screenshot of the rendered content please ? And also whether your using Epytext or restructuredtext.

Thanks

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
The example is on your server. It is about your epytext_demo. You created the [py code](https://github.com/twisted/pydoctor/tree/6e2febe63a8ed0f2e316009f34721dc1778661e7/docs/epytext_demo) and the [HTML outpu](https://pydoctor.readthedocs.io/en/latest/docformat/epytext/epytext_demo.html)t. I just read.

But here you have a screenshot how this is rendered on Debian 11 with Firefox.
![grafik](https://user-images.githubusercontent.com/11861496/167301742-c759344f-d4c5-4b82-aa9c-c3e5069b468f.png)

**Comment by tristanlatr** [comment]
> Current it is: (other than '{' or '}')
    It should look like this: (other than `{` or `}`)

There is no rendering issue with the demo. If you want to improve the documentation by adding inline markup, please feel free to do so.

talk to you later,

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
Again you misunderstood me, and maybe also the intention of the Issue.
I never said this is a rendering issue.

You agree this is a bug in documentation right? It wasn't clear for me.

**Comment by tristanlatr** [comment]
> You agree this is a bug in documentation right? 

No, I don’t think there is a semantical issue in the documentation regarding epytext escaping. Maybe some inline markup could be added to be more explicit.
